polaris-react/src/components/ActionMenu/components/MenuGroup/MenuGroup.tsx

```diff
@@ -22,6 +22,8 @@ export interface MenuGroupProps extends MenuGroupDescriptor {
   getOffsetWidth?(width: number): void;
   /** Collection of sectioned action items */
   sections?: readonly ActionListSection[];
+  /** The element or the RefObject that activates the Modal */
+  groupActivatorRef?: LegacyRef<HTMLSpanElement> | undefined;
```

Comment:
What would be the best place on https://polaris.shopify.com/ to reflect in documentation changes represented in this PR?
I was looking for [Page section](https://polaris.shopify.com/components/layout-and-structure/page) with With custom secondary action, but seems `actionGroups` has type `MenuGroupDescriptor[]` and my new property was added to interface `ComplexAction`